src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -110,6 +110,30 @@ fun displayUserData() {
 
 When you didn't finish the transaction yet, but you start a new Activity, the SDK always automatically finishes the previous Activity transaction. This is due to the fact that only one transaction at a time can be bound to the Scope. To work around that, you may create transactions manually using the custom instrumentation and its instance to start spans instead of the Static API.
 
+### App Start Instrumentation
+
+<Note>
+
+This is a preview API, available in `5.1.0-beta.1` and above. Features available in a preview API are still in progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
+
+</Note>
+
+The App Start Instrumentation provides insight into how long your application takes to launch. It adds a span from the application launch to the first auto-generated UI transaction.
+
+The SDK differentiates between a cold and a warm start, and doesn't track hot starts/resumes.
+* __Cold start__: App launched for the first time, after a reboot or update. The app is not in memory and no process exists.
+* __Warm start__: App launched at least once, is partially in memory, and no process exists.
```

Comment:
As you pointed out in one of our calls, I think Android might cache the process, and it is not created when you start the app. Also, the [Android docs](https://developer.android.com/topic/performance/vitals/launch-time#warm) on warm start:

> The user backs out of your app, but then re-launches it. The process may have continued to run, but the app must recreate the activity from scratch via a call to onCreate().

Reply:
good point, thanks, I borrowed the sentence from the android docs for cold/warm.